src/components/metadata/drag-preview.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const React = require('react')
+const cx = require('classnames')
+const { arrayOf, shape, string } = require('prop-types')
+
+
+class FieldDragPreview extends React.PureComponent {
```

Comment:
We can probably make this a functional component (we've been using `PureComponent` for everything in the past, but nowadays it's probably better to start with functional components for simple components like this one).